Doc/library/pathlib.rst

```diff
@@ -847,10 +847,15 @@ call fails (for example because the path doesn't exist).
    function checks whether *path*'s parent, :file:`path/..`, is on a different
    device than *path*, or whether :file:`path/..` and *path* point to the same
    i-node on the same device --- this should detect mount points for all Unix
-   and POSIX variants.  Not implemented on Windows.
+   and POSIX variants.  On Windows, a mount point is considered to be a drive
+   letter root (e.g. ``c:\``), a UNC share (e.g. ``\\server\share``), or a
+   volume mounted on a filesystem folder.
```

Comment:
Could you change the last case to make it more generic, such as  "a mounted filesystem directory"? I'd prefer to keep it open to a broader interpretation. The phrase "a volume mounted on" to me implies that the entire namespace of the volume is mounted on the directory. That can change, and I think it should change. In POSIX, `ismount()` makes no distinction between a volume mount and a bind mount.